### DISCUSSION TITLE
Generating PDF

### DISCUSSION THREAD

**Discussion by IvanWillsBI** [gotenberg/gotenberg#895] [Q&A] [answered]
I am unable to generate a PDF using a standard HTTP request in Golang, so opted to try and use the unofficial Gotenburg Go client. I have successfully generated a PDF locally, running gotenburg in docker on localhost:3000, but it doesn't work in my production environment as I am unauthenticated.
Doees req.SetBasicAuth(os.Getenv("GOTENBERG_USERNAME"), os.Getenv("GOTENBERG_PASSWORD")) exist in a client?

**Comment by gulien** [comment]
Hello @IvanWillsBI,

You should open an issue or a discussion directly on https://github.com/dcaraxes/gotenberg-go-client!

**Reply by IvanWillsBI** [reply, marked as answer]
Hi @gulien I have done as well - I also just wanted to check in this repo in case anyone else had a workaround
